bosh-errand.sh

```diff
@@ -18,4 +18,4 @@ ${BOSH_PASSWORD}
 EOF
 fi
 
-bosh-cli -n -e env -d "${BOSH_DEPLOYMENT_NAME}" run-errand "${BOSH_ERRAND}"
+bosh-cli -n -e env -d "${BOSH_DEPLOYMENT_NAME}" run-errand "${BOSH_ERRAND}" "${BOSH_FLAGS:-}"
```

Comment:
Great! I think there are a few things I wrote that might needs this sort of check.

https://www.gnu.org/software/bash/manual/bashref.html#Command-Substitution
> `${parameter:-word}`
> If parameter is unset or null, the expansion of word is substituted. Otherwise, the value of parameter is substituted.